cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/stream/FileStreamAdmin.java

```diff
@@ -275,6 +275,12 @@ public void upgrade() throws Exception {
   }
 
   @Override
+  public List<StreamSpecification> listStreams(NamespaceId namespaceId) throws Exception {
+    ensureAccess(namespaceId);
```

Comment:
@bdmogal Ah sorry. I was misled by this 'list	READ/WRITE/ADMIN/ALL (Namespace)' in the wiki. I will do the filtering.